common/persistence/sql/task.go

```diff
@@ -431,27 +431,6 @@ func (m *sqlTaskManager) GetTasks(
 	return response, nil
 }
 
-func (m *sqlTaskManager) CompleteTask(
-	ctx context.Context,
-	request *persistence.CompleteTaskRequest,
-) error {
-	nidBytes, err := primitives.ParseUUID(request.TaskQueue.NamespaceID)
-	if err != nil {
-		return serviceerror.NewUnavailable(err.Error())
-	}
-
-	taskID := request.TaskID
-	tqId, tqHash := m.taskQueueIdAndHash(nidBytes, request.TaskQueue.TaskQueueName, request.TaskQueue.TaskQueueType)
-	_, err = m.Db.DeleteFromTasks(ctx, sqlplugin.TasksFilter{
-		RangeHash:   tqHash,
-		TaskQueueID: tqId,
-		TaskID:      &taskID})
```

Comment:
you can delete this field from sqlplugin.TasksFilter, fix the comment there, and then clean up the sqlplugin implementations to only work with ranges, and delete the non-range queries from them also

Reply:
thanks for this as well - on it